src/utils/getFilteredTodos.ts

```diff
@@ -0,0 +1,39 @@
+import { Todo } from '../types/Todo';
+import { Filters } from '../types/Filters';
+
+type Params = {
+  query: string;
+  selectedFilter: Filters;
+};
+
+export const getFilteredTodos = (
+  todos: Todo[],
+  { query, selectedFilter }: Params,
+) => {
+  let filteredTodos = [...todos];
+
+  filteredTodos = filteredTodos.filter(todo => {
+    switch (selectedFilter) {
+      case Filters.ACTIVE:
+        return !todo.completed;
+
+      case Filters.COMPLETED:
+        return todo.completed;
+
+      default:
+        return todo;
```

Comment:
filter callback should return boolean value. here it's still working, but it would be better to return just `true`